stable/admin/templates/rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if or (not (hasKey .Values.nuodb "addRoleBinding")) .Values.nuodb.addRoleBinding }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: nuodb-kube-inspector
+roleRef:
+  kind: Role
+  name: nuodb-kube-inspector
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: {{ default "nuodb" .Values.nuodb.serviceAccount }}
```

Comment:
Is there a need to use `default` when rendering `.Values.nuodb.serviceAccount`? It has already defaulted in `values.yaml`.

Reply:
I do this in case someone deletes the field from the values file. I'm not sure what is "best practice" in Helm, but if we expect people to edit `values.yaml` then we should be resilient to them deleting fields and explicitly inject the correct default.

Reply:
Normally the users are creating their own values files and specify them when invoking `helm install`. The `default` doesn't harm ... so I'm fine with it.